### DISCUSSION TITLE
Trigger OP backup prior to firing update

### DISCUSSION THREAD

**Discussion by SuperTrollMan** [OctoFarm/OctoFarm#1289] [Feature Request]
Hi, 

I can see it would be beneficial to have OF trigger a backup in OP prior to firing the OP update.
Ideally this would be copied to the OF server and could be browsed and transferred off at a later date, however in the first instance just triggering the backup to the OP instance would be beneficial should an OP update go wrong.

Thanks!

**Comment by NotExpectedYet** [comment]
> Hi,
> 
> I can see it would be beneficial to have OF trigger a backup in OP prior to firing the OP update. Ideally this would be copied to the OF server and could be browsed and transferred off at a later date, however in the first instance just triggering the backup to the OP instance would be beneficial should an OP update go wrong.
> 
> Thanks!

That's not a bad introduction to the feature. 

I did want to do back up management eventually so will keep this in mind cheers.
